com.rlabrecque.steamworks.net/Runtime/Steam.cs

```diff
@@ -260,7 +261,10 @@ public static ESteamAPIInitResult InitEx(uint unIP, ushort usGamePort, ushort us
 					}
 					else {
 						initResult = ESteamAPIInitResult.k_ESteamAPIInitResult_FailedGeneric;
-						OutSteamErrMsg = "[Steamworks.NET] Failed to initialize CSteamAPIContext";
+						if(!string.IsNullOrEmpty(OutSteamErrMsg))
+							OutSteamErrMsg = $"[Steamworks.NET] Failed to initialize CSteamAPIContext: {OutSteamErrMsg}";
```

Comment:
Can this ever get hit? If OutSteamErrMsg is not empty, does that not also mean we'll never get into the branch at L256?